cilium/CFP-22215-mutual-auth-for-service-mesh.md

```diff
@@ -261,15 +366,28 @@ This process will be skipped if the `mesh-auth-mode` config directive is not set
 * Update datapath to inform it that this peer is authenticated (see Data Plane for details)
 * The destination node is responsible for checking that a Pod matching the destination Identity is running on itself, and only allowing the authentication to succeed if it is.
 
-##### Endpoint deletion
+##### Node deletion
+
+* When a node is deleted, entries in other node's auth tables associated with it are deleted also.
 
-* Node with endpoint must clear out Auth table to remove any state that implies authentication occurred for connections with this endpoint
-* Do all other nodes need to clear out their corresponding state also?
```

Comment:
When an endpoint is deleted on one node, do _other_ nodes need to clear out some corresponding auth state? This point appears to be removed without corresponding text being added.

Reply:
Moved this around and added that the GC removes entries associated with endpoints.